dtschema/schemas/property-units.yaml

```diff
@@ -40,7 +40,10 @@ patternProperties:
     $ref: "types.yaml#/definitions/uint32-array"
     description: megahertz
   "-hz$":
-    $ref: "types.yaml#/definitions/uint32-array"
+    anyOf:
+      - $ref: "types.yaml#/definitions/uint32-array"
+      - $ref: "types.yaml#/definitions/uint32-matrix"
```

Comment:
uint32-matrix is superset of uint32-array, so we don't need both.

Not a new issue, but for the scalar cases of these we don't place any constraints to actually constrain them to a single value (i.e. 'maxItems: 1'). I think that can be added automatically.

Reply:
> uint32-matrix is superset of uint32-array, so we don't need both.

I'll make it only uint32-matrix then.

> Not a new issue, but for the scalar cases of these we don't place any constraints to actually constrain them to a single value (i.e. 'maxItems: 1'). I think that can be added automatically.

You mean that all bindings with 'xxx-hz' should have maxItems?

Reply:
> > Not a new issue, but for the scalar cases of these we don't place any constraints to actually constrain them to a single value (i.e. 'maxItems: 1'). I think that can be added automatically.
> 
> You mean that all bindings with 'xxx-hz' should have maxItems?

Yes, but I think we can do that automatically if the schema doesn't define 'maxItems'. The issue is already there with uint32-array type (on all the unit suffixes).